man/systemd.network.xml

```diff
@@ -1985,6 +1985,19 @@
             Defaults to unset.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>AssociatedWith=</varname></term>
+          <listitem>
+            <para>Specifies where the address is associated with. Takes one of <literal>use</literal>,
+            <literal>self</literal>, <literal>master</literal> or <literal>router</literal>.
+            <literal>use</literal> means the address is in use. User space can use this option to
+            indicate to the kernel that the fdb entry is in use. <literal>self</literal> means
+            the address is associated with the port drivers fdb. Usually hardware. <literal>master</literal>
```

Comment:
I cannot understand `associated with the port drivers fdb`.

I am not sure, but maybe, `associated with the port of the fdb device, usually hardware.`

Reply:
>I cannot understand associated with the port drivers fdb.

Means that the address is assoviated with itself . Please see https://www.linux.org/docs/man8/bridge.html

Reply:
I see. Thank you.